index.js

```diff
@@ -139,26 +190,37 @@ const line = []
 
 // ***** Scope & Closures - Question 1 *****
 
+let newLine = []
+
+function takeATicketNumber(line) {
+  for (const element of line) {
```

Comment:
I like that you're using the for ... of loop, but think about how this would scale as you add/remove people from the line!